frontend/src/components/RecordsList.tsx

```diff
@@ -32,7 +32,7 @@ interface IRecordsListProps {
   enableInfiniteScroll?: boolean;
   lazyRecordsQuery: typeof useHookLazyGeneric;
   lazyRecordsQueryAddlVariables?: Record<string, any>;
-  prepareDataForAgGrid: (
+  prepareDataForAgGrid?: (
```

Comment:
This is the function that previously flattens the data for AG Grid to use. Now that data is flattened before reaching the frontend, this prop is now optional. We can't remove it because the Cohorts page uses this to filter the data (see `function prepareCohortDataForAgGrid` in `helpers.tsx`).